matrix_client/client.py

```diff
@@ -59,6 +59,11 @@ class MatrixClient(object):
             the token) if supplying a token; otherwise, ignored.
```

Comment:
Wait. Why do we have a docstring on both the class and on the `__init__` method with subtly different information on arguments? Poking around, most people seem to just not have a docstring on `__init__` methods. I'll open an issue to move all info in `__init__` docstrings to class docstrings.